### PR TITLE
Absurd replace

### PR DESCRIPTION
**Pull request by renatoi**
@src-code @thierryk Removed absurd dependency. :)

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/atomizer.js [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
 var _ = require('lodash');
 var utils = require('./utils');
+var JSS = require('./jss');
```
Comment (src-code) [inline src/atomizer.js]: :smile:

File: src/atomizer.js [hunk 2]
```diff
@@ -763,7 +740,7 @@ Atomizer.replaceConstants = function (str/*:string*/, rtl/*:boolean*/) {
         return str;
     }
 
-    return str.replace(/__start__/g, start).replace(/__end__/g, end);
+    return str.replace(/__START__/g, start).replace(/__END__/g, end);
```
Comment (src-code) [inline src/atomizer.js]: :+1

File: src/atomizer.js [hunk 3]
```diff
@@ -706,18 +691,10 @@ Atomizer.prototype.getCss = function (config/*:AtomizerConfig*/, options/*:CSSOp
...
-    absurd.add(csso);
-    absurd.compile(function(err, result) {
-        /* istanbul ignore if else */
-        if (err) {
-            throw new Error('Failed to compile atomic css:' + err);
-        }
-        content = options.banner + result;
-    }, options);
+    // convert JSS to CSS
+    content = options.banner + JSS.cssoToCss(csso);
```
Comment (src-code) [inline src/atomizer.js]: Function name is a little confusing...what's csso?  CSS object?

**Comment by src-code** [comment]
What about automatic browser prefixing? Were we benefiting from that in Absurd?

**Comment by src-code** [comment]
Addresses issue #101

File: src/atomizer.js (hunk 3 above)
Comment (renatoi) [inline src/atomizer.js]: Yeah, I think I'll change that to jssToCss

**Comment by src-code** [comment]
LGTM

**Pull request merged**